### PR TITLE
refine prompt

### PR DESCRIPTION
**Pull request by mongodben**

### CODE REVIEW

File: app/openai-client.ts [hunk 1]
```diff
@@ -40,21 +42,14 @@ export const ChatGPT = new ChatGPTAPI({
...
-
-    Nudge new users to sign up for MongoDB Atlas. Sign up links should always use a formatted markdown link
-    and the URL query parameter tck=docs_realm.
-
-    Always include a "Sources" section at the end of every response with a
-    list of sources you used. Only cite real links - do not make up citations.
   `,
   completionParams: {
-    temperature: 0.9,
+    temperature: 0.1,
```
Comment (mongodben) [inline app/openai-client.ts:52]: this makes the model less likely to make stuff up

File: app/pages/api/ask.ts [hunk 2]
```diff
@@ -144,21 +145,35 @@ export default async function handler(
...
       ${question}
+      """
+
+      Include a list of "SOURCES" at the bottom of the response.
+      Only use links in the SOURCE_URLs from the conversation.
+      Only include the SOURCE_URL if the CONTENT next to it is used in the answer.
+      Format the links as Markdown links, such as [https://example.com](https://example.com).
+      ONLY include links in context information provided in this conversation section.
+      NEVER use a link not present in the CONTEXT information.
+      NEVER use a link that is not provided in the chat. NEVER make up a link.
```
Comment (mongodben) [inline app/pages/api/ask.ts:164]: for reasons beyond my understanding, including this stuff last helps emphasize it to the LLM.

Comment (leo-generali) [reply app/pages/api/ask.ts:161]: 🤣🤣🤣

File: app/openai-client.ts (hunk 1 above)
Comment (nlarew) [reply app/openai-client.ts:52]: 0.1 seems way too low - I think we want a little creativity for less "direct lookup" questions.

Maybe 0.6 instead?

**Pull request merged**